core/common/connector-core/src/main/java/org/eclipse/edc/connector/core/LocalPublicKeyDefaultExtension.java

```diff
@@ -0,0 +1,89 @@
+/*
+ *  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.core;
+
+import org.eclipse.edc.connector.core.security.LocalPublicKeyServiceImpl;
+import org.eclipse.edc.runtime.metamodel.annotation.Extension;
+import org.eclipse.edc.runtime.metamodel.annotation.Inject;
+import org.eclipse.edc.runtime.metamodel.annotation.Provider;
+import org.eclipse.edc.runtime.metamodel.annotation.Provides;
+import org.eclipse.edc.runtime.metamodel.annotation.Setting;
+import org.eclipse.edc.spi.EdcException;
+import org.eclipse.edc.spi.iam.LocalPublicKeyService;
+import org.eclipse.edc.spi.result.Result;
+import org.eclipse.edc.spi.security.KeyParserRegistry;
+import org.eclipse.edc.spi.security.Vault;
+import org.eclipse.edc.spi.system.ServiceExtension;
+import org.eclipse.edc.spi.system.ServiceExtensionContext;
+import org.eclipse.edc.spi.system.configuration.Config;
+
+import java.util.Map;
+
+import static org.eclipse.edc.connector.core.SecurityDefaultServicesExtension.NAME;
+
+@Extension(value = NAME)
+@Provides(LocalPublicKeyService.class)
+public class LocalPublicKeyDefaultExtension implements ServiceExtension {
+    
+    public static final String NAME = "Local Public Key Default Extension";
+
+    public static final String EDC_PUBLIC_KEYS_PREFIX = "edc.iam.publickeys";
```

Comment:
@Setting

Reply:
It's not needed this is only the prefix the `@Setting` is on `id` and `value`